doc/project/release_process.rst

```diff
@@ -85,24 +85,146 @@ At that point, the whole process starts over again.
 
 .. _release_quality_criteria:
 
-Release Quality Criteria
-************************
+Release Criteria
+****************
+
+The main motivation is to clearly have the criteria in place that must be met for a public releases to ship.
+This will help define when a release is "done" in terms that most people can
+understand and in ways that help new people to understand the process and
+participate in creating successful releases:
+
+- The release criteria documents all the requirements of our target audience for each Zephyr release
+- The target audiences for each public release can be different, and may overlap
+- The criteria at any given time are not set in stone: there may be requirements
+  that have been overlooked, or that are new, and in these cases, the criteria
+  should be expanded to ensure all needs are covered.
+
+Below is the high level criteria to be met for each release:
+
+- No blocker bugs
```

Comment:
i would be tempted to s/bug/issue/ across mostly the whole doc, except for the instances where we do mean "bug". as in a defect in code?
But not sure that will work with e.g. the current definition of blocker below. 
Just thinking about the "blocker" issue we had for 3.3 where we didn't want to ship until the issue with web analytics and cookie consent would be addressed.